README.md

```diff
@@ -1,5 +1,15 @@
+# 2020-1-OSS Team 10
```

Comment:
This is fine, but could you move this block to the bottom of the README? I don't want to take away from the project introduction